graphqlapi/src/main/java/io/stargate/graphql/schema/schemafirst/processor/MutationMappingModelFactory.java

```diff
@@ -67,17 +70,17 @@ private static Kind detectType(FieldDefinition mutation, ProcessingContext conte
    * explicit directive, or otherwise from a set of predefined name prefixes.
    */
   private enum Kind {
-    INSERT(InsertMappingModel::build, "cql_insert", "insert", "create"),
-    UPDATE(UpdateMappingModel::build, "cql_update", "update"),
-    DELETE(DeleteMappingModel::build, "cql_delete", "delete", "remove"),
+    INSERT(InsertMappingModelBuilder::new, "cql_insert", "insert", "create"),
+    UPDATE(UpdateMappingModelBuilder::new, "cql_update", "update"),
+    DELETE(DeleteMappingModelBuilder::new, "cql_delete", "delete", "remove"),
     ;
```

Comment:
why no `SELECT` here?

Reply:
Selects are queries, not mutations. We process queries separately, see `MappingModel.build()`.